src/README.md

```diff
@@ -1,6 +1,5 @@
 This directory contains the source code of the rust project, including:
```

Comment:
This sentence isn't really correct anymore. Most of the source code is in `compiler` and `library`.